src/app/pages/join-session/join-session.component.html

```diff
@@ -10,7 +10,15 @@
         </mat-form-field>
         <mat-form-field appearance="outline" class="form-field">
           <mat-label>Name</mat-label>
-          <input matInput class="name"  formControlName="nameControl">
+          <input matInput class="name" formControlName="nameControl">
+        </mat-form-field>
+        <mat-form-field appearance="outline" class="form-field">
+          <mat-label>Nest marker ID</mat-label>
+          <input matInput inputmode="numeric" pattern="[0-9]*" class="nest" formControlName="nestControl">
```

Comment:
I feel like we could make this pattern more specific if we're going to use it at all. Something like this would match exactly 20-120: `[2-8][0-9]|9[0-9]|1[01][0-9]|120`

Reply:
Also we should mark all the input elements here with the `required` attribute.

Reply:
Having the inputs be required, only allowing numbers in the nest ID field and checking the value is between 20 and 120 is done by Angular Form Controls' Validators. 
`sessionFormGroup = new FormGroup({
    nameControl: new FormControl('', Validators.required),
    sessionControl: new FormControl('', Validators.required),
    nestControl: new FormControl('', [Validators.required, Validators.min(20), Validators.max(120),
      Validators.pattern('^[0-9]*$')])
  });`

 The `pattern="[0-9]*"` is used to bring up the numerical keypad on iOS devices.

Reply:
1. The pattern is not needed for the iOS keyboard, it does that just from the `inputmode`
2. It's still nice to hint to the browser that the field is required.

Reply:
`inputmode `only bring up the numeric keyboard for iOS 12.2+ which was only available after March 2019. Experimenting on browserstack, there are a couple iOS devices that don't bring up the right keyboard just using `inputmode` and need the `pattern` attribute. We might as well support the most devices possible considering elementary/middle school kids often have hand-me downs or older models anyways.

I can see adding a required attribute for the browsers, but I rather keep the pattern the way it is in both Validator & inputmode just because it's more flexible. If we change the range in the future, we can just change the min and max Validator instead of change those and the regex.

Reply:
I would prefer to add the `required` attribute to the inputs if we can.